units.md

```diff
@@ -0,0 +1,83 @@
+# Massa Units
+
+**Authors:**
+**Status:** Draft
+**Version:** 0.1
+
+## Abstract
+
+This specification defines the units used to measure the different objects of the Massa blockchain. It aims to provide a standardized format for the different units to offer the simplest and most readable user experience possible while being consistent from one application to another within our ecosystem.
+
+> _NOTE:_ The manipulation of very large or very small numbers will be simplified by the use of [metric prefixes](https://en.wikipedia.org/wiki/Metric_prefix) of the [SI](https://en.wikipedia.org/wiki/International_System_of_Units).
+
+## Targeted Audience
+
+This specification is intended for developers who wish to interact with the Massa blockchain network using various tools and applications such as node API, smart contract tooling, dApps tooling, Thyra, and Thyra plugins.
+
+## Units
+
+### Massa coin
+
+Massa coin is the name of the native coin on the Massa blockchain.
+Massa coin is expressed as an unsigned 64-bit integer.
+
+1 Massa is equivalent to 1,000,000,000 nanoMassa, the smallest unit of Massa coin.
+
+To simplify usage, the following standard units implemenation are recommended:
+
+- milliMassa (mMASSA): 1 milliMassa represents 1,000,000 nanoMassa and also 1/1000th of a Massa.
+- microMassa (µMASSA): 1 microMassa represents 1,000 nanoMassa and also 1/1,000,000th of a Massa.
+- nanoMassa (nMASSA): 1 nanoMassa represents the smallest unit of Massa coin and also 1/1,000,000,000th of a Massa.
+
+### Gaz
+
+Gaz is a virtual unit used to measure the computational complexity of executing transactions on the Massa blockchain. Each operation requires a certain amount of Gaz, and the amount of Gaz consumed is proportional to the computational complexity of the operation. In order to make this unit more tangible for the end user, gaz is measured in liters (L).
+Gaz is expressed as an unsigned 64-bit integer.
+
+1 Liter (L) is equivalent to 1,000,000,000 nanoliter, the smallest unit of gaz.
+
+To simplify usage, the following standard units implemenation are recommended:
```

Comment:
The litres metrics sounds very weird. Maybe renaming it to MGAS for Massa Gas or something is an option ?

Reply:
great!